src/main/groovy/com/athaydes/spockframework/report/vivid/SpecSourceCode.groovy

```diff
@@ -0,0 +1,29 @@
+package com.athaydes.spockframework.report.vivid
+
+import groovy.transform.ToString
+import org.codehaus.groovy.ast.MethodNode
+import org.spockframework.runtime.model.BlockKind
+
+@ToString
+class SpecSourceCode {
```

Comment:
I see you're using types everywhere, so can you try to make this use `@CompileStatic`?